### PR TITLE
[WIP] Allow enabling/disabling of adlists via webUI

### PR DESCRIPTION
**Pull request by DL6ER**
**By submitting this pull request, I confirm the following (please check boxes, eg [X])Failure to fill the template will close your PR:**

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] I have checked that [another pull request](https://github.com/pi-hole/pi-hole/pulls) for this purpose does not exist.
- [X] I have considered, and confirmed that this submission will be valuable to others.
- [X] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [X] I give this submission freely, and claim no ownership to its content.

**How familiar are you with the codebase?:**

- [X] 10 (very familiar)

---
Backend for https://github.com/pi-hole/AdminLTE/pull/252

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

### CODE REVIEW

**Comment by DL6ER** [comment]
Description of 0a82f45

```
NEW LOGIC FOR ADLISTS
On running the installer (fresh installation, update, ...):
  Copy /etc/.pihole/adlists.default to /etc/pihole/adlists.defaults to enforce out recommendations for this list once per updat$

On running gravity:
  Test if file adlist.list exists, if not: copy over from adlist.defaults
  Collect URLs from adlist.list
  Collect URLs from (new) adlist.user (if exists)
```

**Comment by DL6ER** [comment]
Resolved merge conflicts. Ready for review and merge.

File: gravity.sh [hunk 1]
```diff
@@ -29,6 +29,7 @@ EOM
 
 adListFile=/etc/pihole/adlists.list
 adListDefault=/etc/pihole/adlists.default
```
Comment (Mcat12) [inline gravity.sh]: Needs an `s` at the end

Comment (DL6ER) [inline gravity.sh]: This is interesting, I haven't changed that. This line goes back to a commit from Adam in January 352ebc3193d17ae4e075eec7f71bfe909ab8fa06

Comment (PromoFaux) [inline gravity.sh]: What needs an `s` at the end?

Comment (DL6ER) [inline gravity.sh]: Everything is fine now. Was a misunderstanding between @Mcat12 and @DL6ER

**Comment by DL6ER** [comment]
WIP since we might change the format of the adlists to CSV

**Comment by PromoFaux** [comment]
Made obsolete by #1365

**Pull request closed without merging**